helm/druid/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 apiVersion: v2
-appVersion: 0.19.0
+appVersion: 0.22.0
 description: DEPRECATED Apache Druid is a high performance real-time analytics database.
```

Comment:
What does the 'DEPRECATED' mean ?

Reply:
The Druid Chart was deprecated in [helm/charts](https://github.com/helm/charts/tree/master/incubator/druid) repo
> ⚠️ Repo Archive Notice
> As of Nov 13, 2020, charts in this repo will no longer be updated. For more information, see the Helm Charts Deprecation and Archive Notice, and Update.

The deprecation notice can be removed from this repo. @asdf2014 what do you suggest ?

Reply:
I think we can remove this flag.

Reply:
the flag and message has been removed in #11897